src/NUnitFramework/framework/Internal/InvalidDataSourceException.cs

```diff
@@ -3,7 +3,9 @@
 namespace NUnit.Framework.Internal
 {
     using System;
+#if !NET8_0_OR_GREATER
     using System.Runtime.Serialization;
+#endif
```

Comment:
Just suggestions. Should we move the usings outside the namespace? (and perhaps move `System.Runtime.Serialization`  down to the sole purpose, so we avoid having two `!NET8_0_OR_GREATER` blocks ?

EDIT: I can see that we do it differently in many of the `...Exception` files, so I'm also happy with the current solution (to keep the fix as small as possible)